src/worldofzuul/Game.java

```diff
@@ -12,7 +17,17 @@ public Game() {
         createRooms();
```

Comment:
Constructor methoden skal også have en docblock, der beskriver hvad den gør.

Reply:
fixed in: db92ec45a10ec63c76d3a7c5b241ffd230b09c31